### DISCUSSION TITLE
Don't have the Upgrade option

### DISCUSSION THREAD

**Discussion by adipanait** [NightscoutFoundation/xDrip#2657] [Q&A] [answered]
I searched high and low through all Billing and Credit details- cannot find the Upgrade option. I would like to do this before the Free Trial expires, as indicated in the Documentation:  https://navid200.github.io/xDrip/docs/Nightscout/FreeTrialEnd.html

![image](https://user-images.githubusercontent.com/17168506/216294572-26c253e6-5b0a-44d1-81e3-671062b41a63.png)

please help.

**Comment by Navid200** [comment, marked as answer]
Let's go one step at a time.
First:  Click on this:
![216294572-26c253e6-5b0a-44d1-81e3-671062b41a63](https://user-images.githubusercontent.com/51497406/216343754-5d8d72af-89a6-402c-9790-6d04b64150c7.png)

**Reply by adipanait** [reply]
I get to this: 
![image](https://user-images.githubusercontent.com/17168506/216378692-e9e25f48-2042-438e-9115-b68d4051bced.png)

**Reply by Navid200** [reply]
According to that, you have already upgraded.

**Reply by adipanait** [reply]
oh, so then - the warning next to Free Trial (that says 5 days left) - is just for my info? that's great. Wish they could have made it clearer.

**Reply by Navid200** [reply]
You have $299 credit from your free trial.  But, you only have 5 days left to use that credit.
I see the cause of the confusion.
Thanks.

I will add a note to the page to help others.

**Reply by adipanait** [reply]
Thank you!
